index.js

```diff
@@ -1,13 +1,31 @@
-module.exports = one;
-module.exports.all = all;
+'use strict';
 
-function one (selector, parent) {
-  parent || (parent = document);
-  return parent.querySelector(selector);
+function getElementsArray (elements) {
+	// some elements are array-like (<form> has a .length) so they need to be stopped here
+	if (elements instanceof HTMLElement || elements === document) {
+		return [elements];
+	}
+	return Array.prototype.slice.call(elements || []);
 }
 
-function all (selector, parent) {
-  parent || (parent = document);
-  var selection = parent.querySelectorAll(selector);
-  return  Array.prototype.slice.call(selection);
-}
+module.exports = function (selector, parent) {
+	return (parent || document).querySelector(selector);
+};
+
+var current, i, l, ii, ll;
```

Comment:
I'm guessing this is a performance optimization, but I still don't like the idea of global variables.